#controller_configuration:
  #axis_linear_z_inc: 4
  #axis_linear_z_dec: 5
  #axis_linear_y: 0
  #axis_linear_x: 1
  #axis_angular_yaw: 3
  #axis_angular_pitch: 4
  #btn_angular_roll_inc: 0
  #btn_angular_roll_dec: 1
  #btn_close_gripper: 2
  #btn_open_gripper: 3
controller_configuration:
  translate_x:
    scale: 0.5
    button_inc_index: 4
    button_dec_index: 5
  translate_y:
    axis_index: 0
    scale: -1.0
  translate_z:
    axis_index: 1
    scale: -1.0
  rotate_roll:
    axis_index: 3
    scale: 2.0
  rotate_pitch:
    axis_index: 4
  rotate_yaw:
    button_inc_index: 1
    button_dec_index: 0
  gripper:
    button_inc_index: 2
    button_dec_index: 3
  